File: WeatherReport/WeatherReport.Web/Models/Weather.cs
namespace WeatherReport.Web.Models
{
    public class Weather
    {
        public string Country { get; set; }
        public string CountryId { get; set; }
        public City City { get; set; }
        public string Time { get; set; }
        public string Wind { get; set; }
        public string Visibility { get; set; }
        public string SkyConditions { get; set; }
        public string Temperature { get; set; }
        public string DewPoint { get; set; }
        public string RelativeHumidity { get; set; }
        public string Pressure { get; set; }
    }
}

File: WeatherReport/WeatherReport.Web/client/src/app/models/Weather.ts
import { City } from './City';

export class Weather {
    constructor(
      public Country: string,
      public City: City,
      public subBannnerImageUrl: string,
      public Time : string,
      public Wind : string,
      public Visibility : string,
      public SkyConditions : string,
      public Temperature : string,
      public DewPoint : string,
      public RelativeHumidity : string,
      public Pressure : string,
    ) {

    }
}  